applications/GeoMechanicsApplication/custom_elements/transient_Pw_line_element.h

```diff
@@ -150,16 +150,30 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) TransientPwLineElement : public Elem
     {
         KRATOS_TRY
 
-        CheckUtilities::CheckDomainSize(GetGeometry().DomainSize(), Id(), "Length");
+        const auto element_Id = Id();
+        CheckUtilities::CheckDomainSize(GetGeometry().DomainSize(), element_Id, "Length");
         const auto r_geometry = GetGeometry();
         CheckUtilities::CheckHasNodalSolutionStepData(
             r_geometry,
             {std::cref(WATER_PRESSURE), std::cref(DT_WATER_PRESSURE), std::cref(VOLUME_ACCELERATION)});
         CheckUtilities::CheckHasDofs(r_geometry, {std::cref(WATER_PRESSURE)});
-        CheckProperties();
+
+        const PropertiesType& r_properties = GetProperties();
```

Comment:
I'm not sure whether I'd prefer to define a separate alias for the element properties. Inlining the function call would be equally clear and save one name. But if you would like to use a dedicated alias, could you please use `auto` rather than repeating the type name?
```suggestion
        const auto& r_properties = GetProperties();
```

Reply:
Sorry, it was a copy-and-paste from another place. Now I went through all of them and put `auto`. In some places like this one used GetProperties() in function calls.